Source/Fuse.Triggers/Trigger.uno

```diff
@@ -16,6 +16,11 @@ namespace Fuse.Triggers
 		Never,
 		/** Only changes during the rooting frame are handled as bypass, without special exceptions. */
 		Rooting,
+		
+		/** Deprecated: 2017-07-21
+			For possible backwards compatibilty, like Standard but excludes the check for layout bypass.
+			This mode should not be used. */
+		ExceptLayout,
```

Comment:
@mortoray: Ugh, yeah. How about assigning it a special value, and checking for that special value or something? Sounds a bit nasty, though...

It's not a big deal, though. We can mark this as "harder" deprecated in a follow-up if we figure out a way that doesn't suck...